pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -531,6 +531,17 @@ containing the generator terminals used as angle reference in the load flow calc
 There is one Terminal created/added in the extension for each calculated Synchronous Component.
 Works only when `referenceBusSelectionMode` is set to `GENERATOR_REFERENCE_PRIORITY`.
 
+**voltageTargetPriorities**  
+When multiple equipment regulate the same bus with different voltage targets,
+this parameter enables configuring priority to resolve inconsistencies by aligning the voltage targets.
+Priority is determined by equipment type order; the voltage target of the equipment type listed first takes precedence over those listed later. 
+By default, the order is `["GENERATOR", "TRANSFORMER", "SHUNT"]`.  
+Note that `"GENERATOR"` indistinctively includes generators, static var compensators, and VSC HVDC converters.
```

Comment:
And batteries?

Reply:
If I am not mistaken today batteries in iIDM are not able to perform voltage control.

Reply:
https://github.com/powsybl/powsybl-open-loadflow/pull/1004